src/cljs/reviews_next/db.cljs

```diff
@@ -1,4 +1,7 @@
 (ns reviews-next.db)
 
 (def initial-db
-  {})
+  {:review_event_title "Hello"
```

Comment:
Nest the page state under a key for the page. 

You can also look at name spaced keys to avoid clashes in the state, since it's likely that one or more pages  contain a similarly named state key.